packages/contracts/docs/osx/01-how-it-works/01-core/03-plugins/index.md

```diff
@@ -17,4 +17,41 @@ Plugins can be related to:
 
 - And **anything** else that comes to mind!
 
-In the next section, we will learn more about plugins, how they work, and who owns and manages them.
+## Understanding Plugins
+
+Plugins exist as instances created from the same base contract by
+
+- copying
+- cloning
+- create proxy
+
+and **each instance is attached to and managed by exactly one DAO**.
```

Comment:
This is technically not true. A DAO could decide to follow an anti-pattern, even if it doesn't make sense to do so.

Reply:
```suggestion
```